Gemfile

```diff
@@ -12,6 +12,7 @@ gem "uglifier", ">= 1.3.0"
 gem "coffee-rails", "~> 4.1.0"
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
+gem 'haml-rails'
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

<!-- Comment by [SideCI](https://sideci.com) -->